src/peer_discovery.cpp

```diff
@@ -247,6 +245,37 @@ PeerDiscovery::DomainPeerDiscovery::reloadMessages()
     }
 }
 
+void
+PeerDiscovery::DomainPeerDiscovery::reDiscover()
+{
+	asio::error_code ec;
+
+	sockFd_.set_option(asio::ip::multicast::leave_group(sockAddrSend_.address()), ec);
+	if (ec)
```

Comment:
if (ec and logger_)

indentation

Reply:
i followed line 163+164's logic. but yes ```if (ec and logger_)``` will catch the exception also.
anyway, do you want this to be changed also?
from
```
if (ec and ec != asio::error::operation_aborted)
    if (logger)
```
to
```
if (ec and (ec != asio::error::operation_aborted) and logger)
```